src/components/NotificationListItem.cy.tsx

```diff
@@ -0,0 +1,120 @@
+import { mount } from "@cypress/react";
+import NotificationListItem from "./NotificationListItem";
+import { PaymentNotificationStatus } from "../../src/models";
+
+describe("Notification List Item", () => {
+  it("displays the correct username and message when a payment is received.", () => {
+    const notification = {
+      userFullName: "Kaylin Homenick",
+      id: "95qRwfJnSORQ",
+      uuid: "0b5a0f25-c56b-4434-ba4d-21f133d8964f",
+      userId: "t45AiwidW",
+      transactionId: "xAsSYDsiEGSj",
+      status: PaymentNotificationStatus.received,
+      isRead: false,
+      createdAt: new Date(),
+      modifiedAt: new Date(),
+    };
+
+    const updateNotification = cy.spy().as("updateNotificationSpy");
```

Comment:
@robertguss I think the alias can be removed here as I don't see it used.

Reply:
updated. Nice catch.